src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -0,0 +1,222 @@
+        this.uploadAvatarPromise = uploadAvatar(image).then(url => new Promise((resolve) => {
+            this.setState({avatarURL: url}, resolve);
+        })).catch(() => {
+            Growl.error(this.props.translate('workspace.editor.avatarUploadFailureMessage'));
+        }).finally(() => updateLocalPolicyValues(this.props.policy.id, {isAvatarUploading: false}));
+    }
+
+    onImageRemoved() {
+        this.setState({previewAvatarURL: '', avatarURL: ''});
+    }
+
+    /**
+     * @returns {Object[]}
+     */
+    getCurrencyItems() {
+        const currencyListKeys = _.keys(this.props.currencyList);
+        return _.map(currencyListKeys, currencyCode => ({
+            value: currencyCode,
+            label: `${currencyCode} - ${this.props.currencyList[currencyCode].symbol}`,
+        }));
+    }
+
+    submit() {
+        updateLocalPolicyValues(this.props.policy.id, {isPolicyUpdating: true});
+
+        // Wait for the upload avatar promise to finish before updating the policy
+        this.uploadAvatarPromise.then(() => {
+            const name = this.state.name.trim();
+            const avatarURL = this.state.avatarURL;
+            const policyID = this.props.policy.id;
+            const currency = this.state.currency;
+
+            update(policyID, {name, avatarURL, outputCurrency: currency});
+        }).catch(() => {
+            updateLocalPolicyValues(this.props.policy.id, {isPolicyUpdating: false});
+        });
+    }
+
+    render() {
+        const {policy} = this.props;
+
+        if (!Permissions.canUseFreePlan(this.props.betas)) {
+            console.debug('Not showing workspace editor page because user is not on free plan beta');
+            return <Navigation.DismissModal />;
+        }
+
+        if (_.isEmpty(policy)) {
+            return null;
+        }
+
+        const isButtonDisabled = policy.isAvatarUploading
+                                  || (this.state.avatarURL === this.props.policy.avatarURL
+                                    && this.state.name === this.props.policy.name);
+        return (
+            <WorkspacePageWithSections
+                headerText={this.props.translate('workspace.common.edit')}
+                route={this.props.route}
+            >
+                {hasVBA => (
+                    <>
+                        <View style={[styles.pageWrapper, styles.flex1, styles.pRelative]}>
+                            <View style={[styles.w100, styles.flex1]}>
+                                <AvatarWithImagePicker
+                                    isUploading={policy.isAvatarUploading}
+                                    avatarURL={this.state.previewAvatarURL}
+                                    size={CONST.AVATAR_SIZE.LARGE}
+                                    DefaultAvatar={() => (
+                                        <Icon
+                                            src={Camera}
```

Comment:
This should be `Workspace` right? The import at the top of the file for `Camera` also needs to change.

It looks like this right now:
![image](https://user-images.githubusercontent.com/19364431/136435277-8cde54f5-8bbf-4b0a-add7-ac5a326cfc37.png)

And I think the doc has it looking like this here:
![image](https://user-images.githubusercontent.com/19364431/136435586-656bedc7-18b8-4b5c-9799-16fc8e4df873.png)

Reply:
Ah, OK. Yeah, I'll fix that. I missed it earlier